examples/context/multiple-contexts.js

```diff
@@ -0,0 +1,47 @@
+// Theme context, default to light theme
+// highlight-next-line
+const ThemeContext = React.createContext('light');
+
+// Signed-in user context
+// highlight-next-line
+const UserContext = React.createContext();
+
+// An intermediate component that depends on both contexts
+const Toolbar = props => {
+  // highlight-range{2-10}
+  return (
+    <ThemeContext.Consumer>
+      {theme => (
+        <UserContext.Consumer>
+          {user => (
+            <ProfilePage user={user} theme={theme} />
+          )}
+        </UserContext.Consumer>
+      )}
+    </ThemeContext.Consumer>
+  );
+};
+
+class App extends React.Component {
+  static propTypes = {
```

Comment:
This is experimental syntax. Please don't use it in stable docs. I'd also say propTypes aren't necessary for this doc.